PerformanceStatus.fsh

```diff
@@ -41,9 +43,10 @@ Description:    "The Eastern Cooperative Oncology Group (ECOG) Performance Statu
 * basedOn only Reference(ServiceRequest | CarePlan)
 * partOf only Reference(Procedure)
 * category from http://hl7.org/fhir/ValueSet/observation-category (extensible)
-* code = LNC#89262-0 "ECOG Performance Status [Interpretation]"
+* code = LNC#89247-1 "ECOG Performance Status score"
 * subject only Reference(USCorePatient)
 * effective[x] only dateTime or Period 
 * performer only Reference(Practitioner)
-* value[x] only CodeableConcept
-* valueCodeableConcept from ECOGPerformanceStatusVS (required)
+* value[x] only integer
+* interpretation from http://loinc.org/vs/LL529-9 (extensible)
+// * valueCodeableConcept from ECOGPerformanceStatusVS (required)
```

Comment:
You must remove ECOGPerformanceStatusVS from the value set file as well, or it will be included in the IG.